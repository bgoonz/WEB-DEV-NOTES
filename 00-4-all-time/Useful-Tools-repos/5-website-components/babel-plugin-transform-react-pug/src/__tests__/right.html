<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html>

<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>__tests__</title>
<style type="text/css">
HTML {font-size: small; font-family: sans-serif}
BODY {background: #F8F8FA}
A {text-decoration: none}
A:hover {color: #FFF; background: #006699; border-radius: 8px; padding-left: 3px; padding-right: 3px;}
TABLE.fancy {background: #DFDFDF; border-left: 1px solid;
   border-top: 1px solid; border-bottom: 0px solid; border-color: #FFF}
TABLE.fancy TD {border-right: 1px solid; border-bottom: 1px solid;
  border-color: #FFF; text-align: center; padding-right: 5px}
TABLE.fancy TD.no-right {border-right: 0px}
TABLE.fancy TD.right {text-align: right}
TABLE.fancy TD.left {text-align: left}
TABLE.fancy TD.center {text-align: center}
TABLE.fancy TH {border-right: 1px solid; border-bottom: 1px solid; text-align: center;
   background: #488CB1; color: #FFF; border-color: #FFF; font-weight: bold}
.anchor {font-weight: bold; border-top: 0px solid; text-align: center;
   border-bottom: 1px solid; background: #006699; color: #FFF}
.dark-blue {font-weight: bold; border-top: 0px solid; text-align: center;
   border-bottom: 1px solid; background:#00537D; color: #FFF}
.no-top-border {border-top: 0px}
.centered {margin: auto}
.rZ {text-align: right}
.row2 {background: #D3D3E1}
</style>
</head>

<body>

<table border="0" cellspacing="0" cellpadding="3" summary="file table" class="fancy centered">
<tr><td colspan="16" class="dark-blue" style="text-align: center"><b>Folders</b></td></tr>
<tr><td class="left"><a href="../right.html" target="rframe">&lt;parent&gt;</a></td><td class="left">&nbsp;<a href="__snapshots__/right.html" target="rframe">__snapshots__</a></td></tr>
</table>
<br>

<table border="0" cellspacing="0" cellpadding="3" summary="file table" class="fancy centered">
<tr><th><b>File</b></th><th><b>File</b></th><th><b>File</b></th><th><b>File</b></th><th><b>File</b></th><th><b>File</b></th><th><b>File</b></th><th><b>File</b></th></tr>
<tr><td class="left"><a href="attributes-shorthand.input.js">attributes-shorthand.input.js</a>&nbsp;</td><td class="left"><a href="html-entities.input.js">html-entities.input.js</a>&nbsp;</td><td class="left"><a href="invalid-interpolated-component-error.test.js">invalid-interpolated-component-error.test.js</a>&nbsp;</td><td class="left"><a href="js-syntax-error.test.js">js-syntax-error.test.js</a>&nbsp;</td><td class="left"><a href="option-class-attribute.input.js">option-class-attribute.input.js</a>&nbsp;</td><td class="left"><a href="plain-js-each.input.js">plain-js-each.input.js</a>&nbsp;</td><td class="left"><a href="react-fragment-inline-js.input.js">react-fragment-inline-js.input.js</a>&nbsp;</td><td class="left"><a href="text.input.js">text.input.js</a>&nbsp;</td></tr>
<tr class="row2"><td class="left"><a href="attributes-shorthand.test.js">attributes-shorthand.test.js</a>&nbsp;</td><td class="left"><a href="html-entities.test.js">html-entities.test.js</a>&nbsp;</td><td class="left"><a href="invalid-interpolation-error.input.js">invalid-interpolation-error.input.js</a>&nbsp;</td><td class="left"><a href="multi-line-plain-text.input.js">multi-line-plain-text.input.js</a>&nbsp;</td><td class="left"><a href="option-class-attribute.test.js">option-class-attribute.test.js</a>&nbsp;</td><td class="left"><a href="plain-js-if.input.js">plain-js-if.input.js</a>&nbsp;</td><td class="left"><a href="react-fragment.input.js">react-fragment.input.js</a>&nbsp;</td><td class="left"><a href="text.test.js">text.test.js</a>&nbsp;</td></tr>
<tr><td class="left"><a href="attributes-transformation.input.js">attributes-transformation.input.js</a>&nbsp;</td><td class="left"><a href="if-else.input.js">if-else.input.js</a>&nbsp;</td><td class="left"><a href="invalid-interpolation-error.test.js">invalid-interpolation-error.test.js</a>&nbsp;</td><td class="left"><a href="multi-line-plain-text.test.js">multi-line-plain-text.test.js</a>&nbsp;</td><td class="left"><a href="plain-js-case.input.js">plain-js-case.input.js</a>&nbsp;</td><td class="left"><a href="plain-js-while.input.js">plain-js-while.input.js</a>&nbsp;</td><td class="left"><a href="react-fragment.test.js">react-fragment.test.js</a>&nbsp;</td><td class="left"><a href="update-const-error.input.js">update-const-error.input.js</a>&nbsp;</td></tr>
<tr class="row2"><td class="left"><a href="attributes-transformation.test.js">attributes-transformation.test.js</a>&nbsp;</td><td class="left"><a href="if-else.test.js">if-else.test.js</a>&nbsp;</td><td class="left"><a href="js-invalid-expression-error.input.js">js-invalid-expression-error.input.js</a>&nbsp;</td><td class="left"><a href="nested-loops.input.js">nested-loops.input.js</a>&nbsp;</td><td class="left"><a href="plain-js-code-multiline.input.js">plain-js-code-multiline.input.js</a>&nbsp;</td><td class="left"><a href="plain-js.test.js">plain-js.test.js</a>&nbsp;</td><td class="left"><a href="tabs.input.js">tabs.input.js</a>&nbsp;</td><td class="left"><a href="update-const-error.test.js">update-const-error.test.js</a>&nbsp;</td></tr>
<tr><td class="left"><a href="attributes-unescaped.test.js">attributes-unescaped.test.js</a>&nbsp;</td><td class="left"><a href="interpolation.input.js">interpolation.input.js</a>&nbsp;</td><td class="left"><a href="js-invalid-expression-error.test.js">js-invalid-expression-error.test.js</a>&nbsp;</td><td class="left"><a href="nested-loops.test.js">nested-loops.test.js</a>&nbsp;</td><td class="left"><a href="plain-js-code.input.js">plain-js-code.input.js</a>&nbsp;</td><td class="left"><a href="react-fragment-each.input.js">react-fragment-each.input.js</a>&nbsp;</td><td class="left"><a href="tabs.test.js">tabs.test.js</a>&nbsp;</td><td class="left"><a href="variable-declaration.input.js">variable-declaration.input.js</a>&nbsp;</td></tr>
<tr class="row2"><td class="left"><a href="each-else.input.js">each-else.input.js</a>&nbsp;</td><td class="left"><a href="interpolation.test.js">interpolation.test.js</a>&nbsp;</td><td class="left"><a href="js-syntax-error.input.js">js-syntax-error.input.js</a>&nbsp;</td><td class="left"><a href="no-expressions-in-class-attr.test.js">no-expressions-in-class-attr.test.js</a>&nbsp;</td><td class="left"><a href="plain-js-each-multiple.input.js">plain-js-each-multiple.input.js</a>&nbsp;</td><td class="left"><a href="react-fragment-if.input.js">react-fragment-if.input.js</a>&nbsp;</td><td class="left"><a href="test-helper.js">test-helper.js</a>&nbsp;</td><td class="left"><a href="variable-declaration.test.js">variable-declaration.test.js</a>&nbsp;</td></tr>
<tr><td class="left"><a href="each-else.test.js">each-else.test.js</a>&nbsp;</td><td class="left"><a href="invalid-interpolated-component-error.input.js">invalid-interpolated-component-error.input.js</a>&nbsp;</td></tr>
</table>

<p style="font-weight: bold; margin-bottom: 0px">
Folders:  2<br>
Files: 50<br>
Size of all files: 14367 K</p>

</body>
</html>