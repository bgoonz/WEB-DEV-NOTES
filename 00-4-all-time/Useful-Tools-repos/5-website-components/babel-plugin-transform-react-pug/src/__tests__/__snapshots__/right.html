<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html>

<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>__snapshots__</title>
<style type="text/css">
HTML {font-size: small; font-family: sans-serif}
BODY {background: #F8F8FA}
A {text-decoration: none}
A:hover {color: #FFF; background: #006699; border-radius: 8px; padding-left: 3px; padding-right: 3px;}
TABLE.fancy {background: #DFDFDF; border-left: 1px solid;
   border-top: 1px solid; border-bottom: 0px solid; border-color: #FFF}
TABLE.fancy TD {border-right: 1px solid; border-bottom: 1px solid;
  border-color: #FFF; text-align: center; padding-right: 5px}
TABLE.fancy TD.no-right {border-right: 0px}
TABLE.fancy TD.right {text-align: right}
TABLE.fancy TD.left {text-align: left}
TABLE.fancy TD.center {text-align: center}
TABLE.fancy TH {border-right: 1px solid; border-bottom: 1px solid; text-align: center;
   background: #488CB1; color: #FFF; border-color: #FFF; font-weight: bold}
.anchor {font-weight: bold; border-top: 0px solid; text-align: center;
   border-bottom: 1px solid; background: #006699; color: #FFF}
.dark-blue {font-weight: bold; border-top: 0px solid; text-align: center;
   border-bottom: 1px solid; background:#00537D; color: #FFF}
.no-top-border {border-top: 0px}
.centered {margin: auto}
.rZ {text-align: right}
.row2 {background: #D3D3E1}
</style>
</head>

<body>

<table border="0" cellspacing="0" cellpadding="3" summary="file table" class="fancy centered">
<tr><td colspan="16" class="dark-blue" style="text-align: center"><b>Folders</b></td></tr>
<tr><td class="left"><a href="../right.html" target="rframe">&lt;parent&gt;</a></td></tr>
</table>
<br>

<table border="0" cellspacing="0" cellpadding="3" summary="file table" class="fancy centered">
<tr><th><b>File</b></th><th><b>File</b></th><th><b>File</b></th><th><b>File</b></th><th><b>File</b></th><th><b>File</b></th><th><b>File</b></th><th><b>File</b></th></tr>
<tr><td class="left"><a href="attributes-shorthand.test.js.snap">attributes-shorthand.test.js.snap</a>&nbsp;</td><td class="left"><a href="html-entities.test.js.snap">html-entities.test.js.snap</a>&nbsp;</td><td class="left"><a href="invalid-interpolated-component-error.test.js.snap">invalid-interpolated-component-error.test.js.snap</a>&nbsp;</td><td class="left"><a href="js-syntax-error.test.js.snap">js-syntax-error.test.js.snap</a>&nbsp;</td><td class="left"><a href="nested-loops.test.js.snap">nested-loops.test.js.snap</a>&nbsp;</td><td class="left"><a href="plain-js.test.js.snap">plain-js.test.js.snap</a>&nbsp;</td><td class="left"><a href="tabs.test.js.snap">tabs.test.js.snap</a>&nbsp;</td><td class="left"><a href="update-const-error.test.js.snap">update-const-error.test.js.snap</a>&nbsp;</td></tr>
<tr class="row2"><td class="left"><a href="attributes-transformation.test.js.snap">attributes-transformation.test.js.snap</a>&nbsp;</td><td class="left"><a href="if-else.test.js.snap">if-else.test.js.snap</a>&nbsp;</td><td class="left"><a href="invalid-interpolation-error.test.js.snap">invalid-interpolation-error.test.js.snap</a>&nbsp;</td><td class="left"><a href="multi-line-plain-text.test.js.snap">multi-line-plain-text.test.js.snap</a>&nbsp;</td><td class="left"><a href="option-class-attribute.test.js.snap">option-class-attribute.test.js.snap</a>&nbsp;</td><td class="left"><a href="react-fragment.test.js.snap">react-fragment.test.js.snap</a>&nbsp;</td><td class="left"><a href="text.test.js.snap">text.test.js.snap</a>&nbsp;</td><td class="left"><a href="variable-declaration.test.js.snap">variable-declaration.test.js.snap</a>&nbsp;</td></tr>
<tr><td class="left"><a href="each-else.test.js.snap">each-else.test.js.snap</a>&nbsp;</td><td class="left"><a href="interpolation.test.js.snap">interpolation.test.js.snap</a>&nbsp;</td><td class="left"><a href="js-invalid-expression-error.test.js.snap">js-invalid-expression-error.test.js.snap</a>&nbsp;</td></tr>
</table>

<p style="font-weight: bold; margin-bottom: 0px">
Folders:  1<br>
Files: 19<br>
Size of all files: 25729 K</p>

</body>
</html>